### PR TITLE
@l2succes => Some tests!

### PR DESCRIPTION
**Pull request by mzikherman**
Just a little one to start!

### CODE REVIEW

**Comment by l2succes** [comment]
Nice! 👍🏾

File: src/apps/loyalty/server/route_handlers.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { NextFunction, Request, Response } from "express"
+
+export function Home(req: Request, res: Response, next: NextFunction) {
```
Comment (l2succes) [inline src/apps/loyalty/server/route_handlers.tsx]: might as well move all the route handlers to this file if we're gonna go that route

Comment (mzikherman) [inline src/apps/loyalty/server/route_handlers.tsx]: Definitely. 🌞 

The reason I started with this really simple example, is that I had an issue with even import'ing the app, it was choking on some Isomorphic Relay stuff (Container not found).

At some point we'll have to figure that out, but that's sort of why I was just trying something simple to start.

File: src/apps/loyalty/__tests__/server.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  })
+})
+
+describe ("Login page", () => {
+  it("renders the login page", () => {
+    let req = { baseUrl: "loyalty", csrfToken: () => "token" } as any
+    let res = { redirect: jest.fn(), send: jest.fn() } as any
+    let next = jest.fn() as any
+    Login(req, res, next)
+    expect(res.send).toHaveBeenCalledWith("test")
```
Comment (mzikherman) [inline src/apps/loyalty/__tests__/server.tsx]: This is where I want to assert a couple things about the string (which is the rendered DOM and CSS/JS), namely, i'll just be making sure some of the form html exists (and is pointing to the right place).

**Comment by mzikherman** [comment]
Ok @l2succes @alloy I think this is good to merge, a basic snapshot test for the login container, and a simple assertion on some rendered content (+ the tiny home route handler).

This gives us 3 different kinds of tests (snapshot, standard unit test, some content assertions), and also thus far bypasses any really thorny stubbing of styled components, isomorphic relay, etc.

I think this might be enough to get us pretty good test coverage, and we'll cross any of those weird testing bridges if/when we get to them.

In the meantime, I think this is good to go, and I'll be adding some more tests as well.

**Comment by l2succes** [comment]
Tests!! Nicely done.

**Pull request merged**